src/olympia/reviewers/views.py

```diff
@@ -327,6 +330,10 @@ def _queue(request, tab):
             order_by = TableObj.default_order_by()
         if order_by is not None:
             params['order_by'] = order_by
+        filter_form = ReviewQueueFilter(request.GET)
+        if due_date_reasons := request.GET.getlist('due_date_reasons'):
+            filters = [Q(**{reason: True}) for reason in due_date_reasons]
```

Comment:
This is a bit ugly, you're sending the `GET` parameter straight to the ORM with no validation.

How about something like (quickly tested, maybe I'm missing something):
```diff
diff --git a/src/olympia/reviewers/forms.py b/src/olympia/reviewers/forms.py
index b9f651d197..e968141a03 100644
--- a/src/olympia/reviewers/forms.py
+++ b/src/olympia/reviewers/forms.py
@@ -594,11 +594,10 @@ class ReviewQueueFilter(forms.Form):
         choices=(), widget=forms.CheckboxSelectMultiple, required=False
     )
 
-    def __init__(self, data, *args, **kw):
+    def __init__(self, data, *args, **kwargs):
         due_date_reasons = list(Version.objects.get_due_date_reason_q_objects().keys())
-        data = data.copy()
-        data.setlistdefault('due_date_reasons', due_date_reasons)
-        super().__init__(data, *args, **kw)
+        kwargs['initial'] = {'due_date_reasons': due_date_reasons}
+        super().__init__(data, *args, **kwargs)
         labels = {reason: label for reason, label in VIEW_QUEUE_FLAGS}
         self.fields['due_date_reasons'].choices = [
             (reason, labels.get(reason, reason)) for reason in due_date_reasons
diff --git a/src/olympia/reviewers/views.py b/src/olympia/reviewers/views.py
index 845a1220be..73cbc6359b 100644
--- a/src/olympia/reviewers/views.py
+++ b/src/olympia/reviewers/views.py
@@ -325,8 +325,12 @@ def queue(request, tab):
             order_by = TableObj.default_order_by()
         if order_by is not None:
             params['order_by'] = order_by
-        filter_form = ReviewQueueFilter(request.GET)
-        if due_date_reasons := request.GET.getlist('due_date_reasons'):
+        filter_form = ReviewQueueFilter(
+            request.GET if 'due_date_reasons' in request.GET else None
+        )
+        if filter_form.is_valid() and (
+            due_date_reasons := filter_form.cleaned_data['due_date_reasons']
+        ):
             filters = [Q(**{reason: True}) for reason in due_date_reasons]
             qs = qs.filter(functools.reduce(operator.or_, filters))
         table = TableObj(data=qs, **params)
```

That way you also have a "true" `form.data`/`form.cleaned_data` that reflects what was passed.

Reply:
kwargs['initial'] = {'due_date_reasons': due_date_reasons}

Reply:
https://github.com/mozilla/addons-server/pull/22558/commits/900a84cce7b71a33d51acd1d24d7a96c60f88090